src/collision.cpp

```diff
@@ -76,7 +76,7 @@ inline f32 truncate(const f32 val, const f32 factor)
 	return truncf(val * factor) / factor;
 }
 
-inline v3f truncate(const v3f vec, const f32 factor)
+inline v3f truncate(const v3f &vec, const f32 factor)
```

Comment:
This is reverting a change made by 2e89529eef23c678414b0a1a60a04a35724d4df5

https://dev.minetest.net/index.php?title=Code_style_guidelines&diff=6508&oldid=6489

Reply:
Huh, I was not aware of that revision. (Frankly I also don't agree with the reasoning - if passing the value directly is more efficient than passing the const ref, surely the compiler can figure that out, especially with LTO or an inline function as in this case - but I don't mind much either way.)